test/library_checker/two_sat.yosupo.test.cpp

```diff
@@ -0,0 +1,30 @@
+#define PROBLEM "https://judge.yosupo.jp/problem/two_sat"
+#include <bits/stdc++.h>
+#include "src/graph/two_sat.hpp"
+using namespace std;
+
+int main() {
+    string tmp;
+    int N, M;
+    cin >> tmp >> tmp >> N >> M;
+    two_sat ts(N);
```

Comment:
```suggestion
    TwoSAT ts(N);
```

SCCのクラス名を変更した場合はこちらの変更もお願いします 🙇